ingress/controllers/nginx/nginx/main.go

```diff
@@ -67,7 +67,9 @@ func NewManager(kubeClient *client.Client) *Manager {
 
 	ngx.sslDHParam = ngx.SearchDHParamFile(config.SSLDirectory)
 
-	ngx.loadTemplate()
+	if err := ngx.loadTemplate(); err != nil {
+		glog.Fatalf("invalid NGINX template: %v", err)
```

Comment:
> so we don't watch the custom template?

nop, we keep using the old template. 

I'll add a watch to the template file to cover:
1. kubectl exec and overwrite template
2. update in the config map (mounted template)

if there's an error log the error and keep using the working version of the template